### DISCUSSION TITLE
Please add my Synology NAS' NVMe read speed

### DISCUSSION THREAD

**Discussion by 007revad** [007revad/Synology_Information_Wiki#21] [General]
If you want your Synology model and it's NVMe read speed added to the [NVMe read speeds per Synology model table](https://github.com/007revad/Synology_Information_Wiki/blob/main/pages/NVMe-speed.md) please report what the "Timing O_DIRECT disk reads" shows for the following command.
```
sudo hdparm -tT --direct /dev/nvme0n1
```
or
```
sudo hdparm -tT --direct /dev/nvme1n1
```

Reply with:

Synology Model: 
NVMe read speed: 
NVMe brand/model: 
E10M20-T, M2D20 or M2D18:

**Comment by SR71HTU** [comment]
Synology Model: RS3617RPxs
NVMe read speed: 2055 MB/sec
NVMe brand/model: Samsung 970 EVO plus 1TB
E10M20-T, M2D20 or M2D18: AliExpress cheap adaptor card (pcie3.0 x4 lane)

**Reply by 007revad** [reply]
Which AliExpress cheap adaptor card did you get?

**Comment by PreOmegazero** [comment]
Synology Model: DS1019+
NVMe read speed: 153.21 MB/sec
NVMe brand/model: Inland Premium 1TB
E10M20-T, M2D20 or M2D18: Built-in M.2 slot

Synology Model: DS1019+
NVMe read speed: 380.53 MB/sec
NVMe brand/model: Intel SSDPEKKF512G8 512GB
E10M20-T, M2D20 or M2D18: Built-in M.2 slot

**Comment by SR71HTU** [comment]
Hey Dave, great for your respons, I wanted to get in touch with you for
other questions.
Can I send you an email later on other matters (ie compatibility pcie with
2 slots, I also posted a question )

The results I got from a single slot Nvme card on ALiExpress, here is the
link:
https://www.aliexpress.com/item/1005005648091407.html?spm=a2g0o.order_list.order_list_main.5.5ffe1802UY9c0S


Op zo 19 jan 2025 om 23:09 schreef Dave Russell ***@***.***>:

**Reply by 007revad** [reply]
That card looks okay and is very cheap. And it will work in any Synology that has a x4 or x8 PCIe slot (and has /usr/syno/bin/synonvme /us/lib/libsynonvme.so.1).

I've seen other cheap AliExpress single NVMe PCie 3.0 x4 cards but they didn't have a heatsink.

**Reply by SR71HTU** [reply]
Would this nVME work in RS3614 ?  Intiguing statement " It will work in any Synology that has a x4 or x8 PCIe slot (and has /usr/syno/bin/synonvme /us/lib/libsynonvme.so.1).".

**Reply by 007revad** [reply]
I just checked DSM 7.1.1 for the RS3614xs+ and it does not have /usr/syno/bin/synonvme or /us/lib/libsynonvme.so.1

So it would only support M.2 SATA drives.

**Comment by SR71HTU** [comment]
Hello Dave,

I was just wondering if anybody had some experience with the following card
?
[image: image.png]

https://www.aliexpress.com/item/1005005576318321.html?spm=a2g0o.cart.0.0.485238dab48U7S&mp=1
( not cheap, can also be found on Amazon.de for double price aka 120 euros
(resellers !)

Seems to have the ASMedia 2812 chip for bifurcation (which is supposed to
be PCI3.0)...
The only downside I see is that the card itself is made on PCI3.0 x 4
lanes**

1) what speeds to expect with a SHR/RAID1 ?  (taking split in the ASM)
2) do you have in database any speeds SHR/RAID1 measured on Syno card
E10M20-T1 (I believe not the same PLX chip ?)
     BTW this Synocard has PCI3.0 x8 physical interface, would you guess
they use x4 for the 10 GBE part and the other 4 lanes for their NVME part ?

[image: image.png]
3) I wonder*** why we don't find any card with ASM/PLX exploiting the full
PCI3.0 x8 physically to push 4 lanes per Nvme slot ?
Would that not be a thing ? Do you see any on the market ?

Last but not east, I am running all these in a RS3617RPxs machine ( Xeon
1230 ) PCI3.0 x8 dual slot
but I like to know if this can also work in my RS3614xs+ ( which has dual
pcie3.0 x8 as well ??
Do you see an issue with the script to run NVME on this machine ( have not
tried yet)


best regards,
Henk





Op zo 19 jan 2025 om 23:09 schreef Dave Russell ***@***.***>:

**Reply by 007revad** [reply]
12 months ago I started researching suitable cheap non-Synology PCIe M2 cards that I could try to get working in a Synology.
https://www.reddit.com/r/synology/comments/18tdli8/using_a_cheap_pcie_m2_card_in_a_synology_nas_part/

I didn't follow up on it because I was planning on moving and thought the delivery from AliExpress would not arrive before I moved. 

Synology's E10M20-T1 has: 
- AQtion AQC107 ethernet controller chip.
- ASMedia ASM2824 PCIe Gen3 switch chip.
- No AHCI SATA controller chip.

Synology's M2D18 has: 
- Microsemi/PMC/IDT PES24T6G2 PCIe Gen2 switch chip
- Marvell 88SE9235 PCIe 2.0 x2 4-port SATA 6 Gb/s controller chip

I don't own a M2D20 but I suspect it would have: 
- ASMedia ASM2824 PCIe Gen3 switch chip.
- No AHCI SATA controller chip.

**Comment by SR71HTU** [comment]
Dave,

I have another adaptor as well, and I will test next weekend, I guess to
get similar results
https://www.aliexpress.com/item/1005002603686315.html?spm=a2g0o.order_list.order_list_main.10.1ecb18021oW9jo

Will upload the results as well.

PS : I test on the RS 3617

Op zo 19 jan 2025 om 23:09 schreef Dave Russell ***@***.***>:

**Reply by 007revad** [reply]
Yes, that's the other AliExpress single NVMe PCie 3.0 x4 card I was thinking of [here](https://www.reddit.com/r/synology/comments/18tdli8/using_a_cheap_pcie_m2_card_in_a_synology_nas_part/) and I'm 99% sure it will work.

**Reply by SR71HTU** [reply]
link " here" doesn't work

**Reply by 007revad** [reply]
https://www.reddit.com/r/synology/comments/18tdli8/using_a_cheap_pcie_m2_card_in_a_synology_nas_part/

**Comment by SR71HTU** [comment]
no chance to get NVME working with scripting ??

Op ma 20 jan 2025 om 12:09 schreef Dave Russell ***@***.***>:

**Reply by 007revad** [reply]
I believe you'd find that the M.2 slots in the RS3614xs+ are SATA only.

**Comment by MiranoVerhoef** [comment]
Synology Model: RS1221+
NVMe read speed: 2080.16 MB/sec ( cached) / 1465.54 MB/sec (Disk)
NVMe brand/model: Samsung 980 1TB NVME
E10M20-T, M2D20 or M2D18: E10M20-T

**Comment by contributor** [comment]
Synology Model: DS1520+
NVMe read speed: 400 MB/sec (cached read is also 400 MB/sec)
NVMe brand/model: Kingston KC3000 2048 GB (SKC3000D/2048G)
E10M20-T, M2D20 or M2D18: -

**Comment by SR71HTU** [comment]
I have installed this card here below in a RS3617RPxs and confirm it works
smoothly, recognizes both NVMe's.
[image: image.png]

[image: image.png]

Op ma 20 jan 2025 om 11:03 schreef Henk Tubbe ***@***.***>:

**Comment by ntowle** [comment]
Synology Model: DS1525+
NVMe read speed: 718.37 MB/sec
NVMe brand/model: WD Red	SN700 1TB

PCIe Info:

    LnkSta:	Speed 8GT/s (ok), Width x1 (downgraded)
